connector_importer/components/importer.py

```diff
@@ -57,25 +57,22 @@ class RecordImporter(Component):
     _name = "importer.record"
     _inherit = ["importer.base.component"]
     _usage = "record.importer"
-    _apply_on = "import.record"
     # log and report errors
     # do not make the whole import fail
     _break_on_error = False
     _record_handler_usage = "odoorecord.handler"
     _tracking_handler_usage = "tracking.handler"
     # a unique key (field name) to retrieve the odoo record
-    # if this key is an external/XML ID, set 'odoo_unique_key_is_xmlid' to True
+    # if this key is an external/XML ID, prefix the name with `xid::` (eg: xid::id)
     odoo_unique_key = ""
-    odoo_unique_key_is_xmlid = False
```

Comment:
This is a breaking change for existing implementation. Should we add a warning somewhere?

Reply:
all odoo_unique_key attributes are set to id which is already taken as an XMLID so all good